modules/kubernetes-addons/main.tf

```diff
@@ -301,6 +301,18 @@ module "kube_prometheus_stack" {
   addon_context = local.addon_context
 }
 
+module "portworx" {
+  count                = var.enable_portworx ? 1 : 0
+  source               = "portworx/portworx-addon/eksblueprints"
+  version              = "0.0.3"
+  helm_config          = var.portworx_helm_config
+  manage_via_gitops    = var.argocd_manage_add_ons
+  addon_context        = local.addon_context
+  irsa_policies        = var.portworx_irsa_policies
+  set_sensitive_values = var.portworx_set_sensitive_values
+  set_values           = var.portworx_set_values
+  chart_values         = var.portworx_chart_values
```

Comment:
There's no need for passing `set_sensitive_values`, `set_values` and `chart_values` separately. These should be part of the `portworx_helm_config`. So it would be passed in as

```
portworx_helm_config = {
  set_values = [
    {
      name = "clusterName"
      value = module.eks_blueprints.eks_cluster_id
      type  = string
    },
    {
      name  = "imageVersion"
      value = "2.11.2"
      type  = string
    },
  ]
}
```